rskj-core/src/main/java/co/rsk/peg/lockingcap/LockingCapStorageProvider.java

```diff
@@ -0,0 +1,13 @@
+package co.rsk.peg.lockingcap;
+
+import co.rsk.bitcoinj.core.Coin;
+import java.util.Optional;
+
+public interface LockingCapStorageProvider {
+
+    Optional<Coin> getLockingCap();
+
+    void setLockingCap(Coin lockingCap);
+
+    void save();
+}
```

Comment:
please add a blank line at the end of the file

Reply:
This file already has a blank line at the end of the file. On GitHub is difficult to see it I don't know why.
<img width="955" alt="image" src="https://github.com/user-attachments/assets/87f649cc-e95a-4c29-8db4-7e04a2a63daf">

Reply:
ohh ok my bad! thanks